index.html

```diff
@@ -13,158 +13,175 @@
-              &nbsp;&nbsp;&nbsp;
-              <li class="li-bg">Javascript</li>
-              &nbsp;&nbsp;&nbsp;
-              <li class="li-bg">Ruby</li>
-              &nbsp;&nbsp;&nbsp;
-            </ul>
-            <button class="btn">See project</button>
+        </section>
+        <section class="s-card">
+          <div id="card">
+            <div class="card1">
+              <h3>Multi-Post Stories <br />Gain + Glory</h3>
+              <ul class="li">
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">HTML</li>
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">CSS</li>
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">Javascript</li>
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">Ruby</li>
+                &nbsp;&nbsp;&nbsp;
+              </ul>
+              <button class="btn">See project</button>
+            </div>
           </div>
-        </div>
-      </section>
-      <section class="s-card">
-        <div id="card">
-          <div class="card1">
-            <h3>Multi-Post Stories <br />Gain + Glory</h3>
-            <ul class="li">
-              &nbsp;&nbsp;&nbsp;
-              <li class="li-bg">HTML</li>
-              &nbsp;&nbsp;&nbsp;
-              <li class="li-bg">CSS</li>
-              &nbsp;&nbsp;&nbsp;
-              <li class="li-bg">Javascript</li>
-              &nbsp;&nbsp;&nbsp;
-              <li class="li-bg">Ruby</li>
-              &nbsp;&nbsp;&nbsp;
-            </ul>
-            <button class="btn">See project</button>
+        </section>
+        <section class="s-card">
+          <div id="card">
+            <div class="card1">
+              <h3>Multi-Post Stories <br />Gain + Glory</h3>
+              <ul class="li">
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">HTML</li>
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">CSS</li>
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">Javascript</li>
+                &nbsp;&nbsp;&nbsp;
+                <li class="li-bg">Ruby</li>
+                &nbsp;&nbsp;&nbsp;
+              </ul>
+              <button class="btn">See project</button>
+            </div>
           </div>
-        </div>
-      </section>
+        </section>
+      </div>
     </section>
     <section class="content-3 center">
       <img class="abt-1" src="imgs/about1.png" />
       <h1 class="clr-blue">About me</h1>
       <p class="content-3-p">
         I can help you build a product , feature or website Look through some of
-        my work and experience! If you like what you see and have a project you
-        need coded, don’t hesitate to contact me.
+        my work and <br class="hide-mob" />
+        experience! If you like what you see and have a project you need coded,
+        <br class="hide-mob" />
+        don’t hesitate to contact me.
       </p>
       <button class="btn btn-2">Get my resume</button>
```

Comment:
- [ ] Please make sure you follow the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=dWWSbacCPQ0hVj82-0)🎨 for styling the button here. This ensures you follow a given design to perfection which is a quality trait for a developer like yourself.
      ![firefox_kz6PEcoelc](https://user-images.githubusercontent.com/104968236/228802829-d7b93b1c-a0d0-4458-8a22-86209b80a4f2.png)
    - You're only missing the `:hover` state please add it.